manifests/init.pp

```diff
@@ -30,66 +30,63 @@
 #   means. e.g. environment variable ChocolateyBinRoot.  Defaults to
 #   'C:\tools\chocolatey.server'
 class chocolatey_server (
-  $port = $::chocolatey_server::params::service_port,
-  $server_package_source = $::chocolatey_server::params::server_package_source,
-  $server_install_location = $::chocolatey_server::params::server_install_location,
-) inherits ::chocolatey_server::params {
+  $service_port          = '80',
+  $server_package_source = 'https://chocolatey.org/api/v2/',
+  $server_install_location = 'C:\tools\chocolatey.server',
+  $chocolatey_server_app_pool_name = 'chocolatey.server',
+  $chocolatey_server_app_port      = $port,
+) {
```

Comment:
Well then there really isn't a reason to be doing params pattern then. Unless there is something I'm not thinking of offhand